ytmusicapi/parsers/playlists.py

```diff
@@ -79,6 +79,12 @@ def parse_playlist_items(results, menu_entries: Optional[List[List]] = None):
             True,
         )
 
+        track_position = None
+        if "index" in data:
+            # similar item nesting to duration above -> both could be refactored into
```

Comment:
As you noted, please use `nav` here to access the index. You can set `none_if_absent` to simply get `None` if the key is missing (which also catches the case of any subkeys under "index" missing).